frame/support/src/lib.rs

```diff
@@ -946,3 +949,965 @@ mod tests {
 		})
 	}
 }
+
+/// prelude to be used alongside pallet macro, for ease of use.
+pub mod pallet_prelude {
+	pub use sp_std::marker::PhantomData;
+	pub use frame_support::traits::{Get, Interface, IsType, GetPalletVersion};
```

Comment:
```suggestion
	pub use frame_support::traits::{
		BalanceStatus, Currency, EnsureOrigin, ExistenceRequirement, Get, GetPalletVersion,
		Interface, IsType, Randomness, ReservableCurrency, WithdrawReasons
	};
```

Reply:
Ensure origin makes sense to me, but IMO all balances related trait are not exactly central in frame.

Reply:
What do you think about `Randomness`?

Reply:
ah yes, I would tend not to include for same reason as Currency